### PR TITLE
Fix lint errors by adding `tools:ignore="NewApi"` 

### PR DESCRIPTION
**Pull request by larsgrefer**
see also: https://github.com/pockethub/PocketHub/commit/9206195f18addfcc98061aa53755df32c7a085d2#commitcomment-13745222

Additionally this will remove `abortOnError false` in order to get warned by travis, if a pull request introduces new errors

### CODE REVIEW

**Comment by Meisolsson** [comment]
@fadils Anything you want a hand in?

File: app/src/main/res/layout/commit_diff_line.xml [hunk 1]
```diff
@@ -14,14 +14,17 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+<TextView
     android:id="@+id/tv_diff"
     style="@style/CodeText"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
```
Comment (fadils) [inline app/src/main/res/layout/commit_diff_line.xml]: Nit stuff. Could you please place the namespace on the beginning?

File: app/src/main/res/layout/nav_dialog.xml [hunk 2]
```diff
@@ -15,8 +15,9 @@
   limitations under the License.
 -->
 <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content" >
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            xmlns:tools="http://schemas.android.com/tools">
```
Comment (fadils) [inline app/src/main/res/layout/nav_dialog.xml]: Same nit stuff as previous.

**Comment by larsgrefer** [comment]
I've fixed the attribute order

**Comment by larsgrefer** [comment]
@fadils @Meisolsson Is now enverything good here?

**Comment by larsgrefer** [comment]
I've rebased this on the current master

File: app/src/main/res/layout/collaborator_item.xml [hunk 3]
```diff
@@ -40,6 +41,7 @@
         android:paddingLeft="10dp"
         android:paddingStart="10dp"
         android:singleLine="true"
-        android:textStyle="normal" />
+        android:textStyle="normal"
+        tools:ignore="NewApi"/>
```
Comment (hzsweers) [inline app/src/main/res/layout/collaborator_item.xml]: Why is this one necessary?

File: app/src/main/res/layout/contributor_item.xml [hunk 4]
```diff
@@ -44,7 +46,8 @@
             android:paddingLeft="10dp"
             android:paddingStart="10dp"
             android:singleLine="true"
-            android:textStyle="normal" />
+            android:textStyle="normal"
+            tools:ignore="NewApi"/>
```
Comment (hzsweers) [inline app/src/main/res/layout/contributor_item.xml]: Same here. Is textStyle really an API error?

File: app/src/main/res/layout/dashboard_issue_item.xml [hunk 5]
```diff
@@ -32,7 +33,8 @@
         android:layout_toEndOf="@id/ll_issue_number"
         android:paddingLeft="10dp"
         android:paddingStart="10dp"
-        android:singleLine="true" />
+        android:singleLine="true"
+        tools:ignore="NewApi"/>
```
Comment (hzsweers) [inline app/src/main/res/layout/dashboard_issue_item.xml]: single line also shouldn't be an API error

**Comment by hzsweers** [comment]
Ignore my initial comments. Looking at this, we don't need to ignore each case individually and turn off abort on error. We can have a custom lint that tells it to ignore paddingStart. It's annoying that this is still linted though because google resolved those issues some time back with build tools 22, which creates copies of layouts in -v17 directories and removes start/end from the normal ones

**Comment by hzsweers** [comment]
Going to open a separate PR with the custom lint

**Pull request closed without merging**